samples/matter/light_switch/boards/nrf7002dk_nrf5340_cpuapp_release.conf

```diff
@@ -0,0 +1,8 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Do not enable RAM power-down library for nRF7002 DK due to wrong WiFi configuration.
```

Comment:
Hmm.. should we just say `due to incompatible Wi-Fi configuration` not sure if we can say it's wrong.

Reply:
Done.